theme.park.tera

```diff
@@ -1,30 +1,37 @@
+---
+whiskers:
+  version: "2.3.0"
+  matrix:
+    - flavor
+  filename: "themes/{{ flavor.identifier }}.css"
+---
 :root {
-  --main-bg-color: #{{base}};
+  --main-bg-color: #{{ base.hex }};
 
-  --modal-bg-color: #{{mantle}};
-  --modal-header-color: #{{crust}};
-  --modal-footer-color: #{{crust}};
+  --modal-bg-color: #{{ mantle.hex }};
+  --modal-header-color: #{{ crust.hex }};
+  --modal-footer-color: #{{ crust.hex }};
 
-  --drop-down-menu-bg: #{{surface0}};
+  --drop-down-menu-bg: #{{ surface0.hex }};
 
-  --button-color: #{{surface0}};
-  --button-color-hover: #{{surface1}};
+  --button-color: #{{ surface0.hex }};
+  --button-color-hover: #{{ surface1.hex }};
   --button-text: var(--text);
   --button-text-hover: var(--text-hover);
 
   --accent-color: 137, 180, 250; /* Mocha Blue */
```

Comment:
I won't change this for now from the previous template but... why is this always the same mocha blue?

Reply:
I feel like we discussed this at some point but it's just escaping me, maybe @lewisakura can remember?

Reply:
i would like to say it was for compatibility reasons between color schemes and apps but i don't quite remember either, i can't find any conversation history on it so i can't really tell you why

i don't see any particular reason why it can't be changed to the rgb values for blue in each theme, though

Reply:
Shall I change it in this PR then @sgoudham?

Reply:
Sounds good :+1: